sigridci/sigridci/platform.py

```diff
@@ -43,6 +47,8 @@ def getPlatformId():
             return "azure-devops"
         elif Platform.isBitBucket():
             return "bitbucket"
+        elif Platform.isJenkins():
```

Comment:
Just a remark: this method looks for `BUILD_NUMBER`. That's rather generic: I wouldn't be surprised if one of the other platforms als sets that exact same env var (now or in the future). Consequently, order is important here: if we discover `BUILD_NUMBER` but we haven't concluded it is any of the other platforms, then we conclude it must be Jenkins. That's probably exactly what we want, but we might want to add a comment to remind our future selves of that.

Reply:
Good one, added a comment. It was indeed intentional to check Jenkins last, so future platforms should be checked *before* Jenkins.